Source/NSApplication.m

```diff
@@ -1689,6 +1689,7 @@ - (NSModalSession) beginModalSessionForWindow: (NSWindow*)theWindow
 	  [theWindow makeMainWindow];
 	}
     }
+  [theWindow orderFrontRegardless];
```

Comment:
This change looks reasonable to me. Any thoughts @fredkiefer ?

Reply:
The code change looks OK. I have not tested the behaviour but if you have feel free to merge.

Reply:
I have tested this on Debian 12 and it works properly there.

Reply:
@fredkiefer per your comment, I am approving this so that @williameveretteggplant can merge.